### DISCUSSION TITLE
Spanish (Español)

### DISCUSSION THREAD

**Discussion by zsoerez** [sovenor/bitcoin-rocks#24] [Translations]
¡Hola! 
Empecemos a regalar el entendimiento de Bitcoin a nuestros amigos hispanohablantes. Qué emocionante es empezar esta jornada. 

Gracias por cualquier contribución :)

**Comment by ObjectifMoon21** [comment]
!Bienvenido a la traducción español! 

Vamos a compartir aquí los archivos en los que estamos trabajando en este momento, si quieres participar en ese projecto, no dudes en dejar un mensaje aquí y decir en qué archivos vas a trabajar.

**Comment by zsoerez** [comment]
i've begun translating the following:

<img width="300" alt="Captura de pantalla 2024-05-21 a la(s) 5 09 54 p m" src="https://github.com/sovenor/bitcoin-rocks/assets/170360363/41b2075e-0f64-40f0-8bdb-3fe2deb69f61">
<img width="309" alt="Captura de pantalla 2024-05-21 a la(s) 5 08 59 p m" src="https://github.com/sovenor/bitcoin-rocks/assets/170360363/dda21977-2cc6-4fc6-bcb0-9aace28801cf">
<img width="303" alt="Captura de pantalla 2024-05-21 a la(s) 5 08 02 p m" src="https://github.com/sovenor/bitcoin-rocks/assets/170360363/b0019772-a73e-42dc-a40b-271a396d77c7">

**Reply by zsoerez** [reply]
from the first picture, i've translated all of those files, but from the 2nd n 3rd pictures, I have only begun the highlighted files

**Reply by sovenor** [reply]
Amazing, thank you so much for your work on this!

**Reply by zsoerez** [reply]
all of these are done now

**Comment by zsoerez** [comment]
<img width="372" alt="Captura de pantalla 2024-05-24 a la(s) 3 53 11 p m" src="https://github.com/sovenor/bitcoin-rocks/assets/170360363/c2201518-a871-422f-af17-97094abba0ad">

highlighted files are next

**Reply by ObjectifMoon21** [reply]
Great job Z !

**Comment by sovenor** [comment]
Hi, please note that there are 4 new translation strings that need to be added to "common_es.json"

"common_self_custody": "SELF-CUSTODY",
"common_not_your_keys": "NOT YOUR KEYS",
"common_cold_wallet": "COLD WALLET",
"common_hot_wallet": "HOT WALLET"

You can add these to the end of the file. Thanks!

**Comment by sovenor** [comment]
Hi @zsoerez I just wanted to check in. Are you still working on this translation?

**Reply by zsoerez** [reply]
yes! about finished

**Reply by sovenor** [reply]
Amazing, I am really looking forward to your pull request! Let me know if you need help submitting that for review.

**Comment by ObjectifMoon21** [comment]
Hi @zsoerez do you want some help for the translation ? on which file did you stop ?

**Comment by ObjectifMoon21** [comment]
Hi!
Today I started translating the following pages into Spanish :

`404_es.json`
`bank-runs_es.json`
`bitcoin-vs-cbdc_es.json`
`bitcoin-vs-gold_es.json`

I'll continue tomorrow.

**Reply by sovenor** [reply]
Amazing, thank you!

**Comment by ObjectifMoon21** [comment]
¡La versión en español de [bitcoin.rocks](https://bitcoin.rocks/) ya está disponible ! 🎉

Si hay algún hablante nativo que pueda leerla, revisarla y darme comentarios sobre errores o cambios que se deban hacer, ¡sería genial!

**Comment by ObjectifMoon21** [comment]
Hi @sovenor how are you ? 

While i'm working on the proofread and typo, i realise the following new strings are missing in the "es" folder, can you add it so I can translate it for my next pull request: 

Line 11: "inflation_sign_got_inflation"
Line 12: "inflation_save_in_bitcoin"
Line 42: "inflation_usd_but_why"

Also the sign campaign pages are missing, it might be interesting to translate them into spanish for the hispanic community in the united states.

signs_es.json
sign-success_es.json

**Reply by sovenor** [reply]
Yes, these have been added. Thank you for pointing this out, and apologies for the delay in getting back to you!

**Comment by sovenor** [comment]
Translation strings have been modified slightly in wallets_es.json:

Line 11: "two simple questions" is now "these simple questions"
Line 13: "two simple questions" is now "these simple questions"

These new strings have also been added to the same file:

Lines 33 - 39 to add a section about recovery phrases / seed phrases.

Please translate these if you have some spare time. Thank you!

**Reply by ObjectifMoon21** [reply]
Translation of the new strings in spanish are done. PR sent !
